### PR TITLE
docs: installation guide

### PR DESCRIPTION
**Pull request by dibrinsofor**
fixes: #19 
Release Notes:

Type of Changes:
- [X] Improved docs

**Description of changes**: Updated docs to include `go install` plus added mini tutorial for adding GOPATH to path env vars.

### CODE REVIEW

**Comment by dibrinsofor** [comment]
Can I include an installation/usage gif to the docs?

File: README.md [hunk 1]
```diff
@@ -3,12 +3,28 @@
...
+If you have not already added your `GOPATH/bin` directory to your `PATH` environment variable then you will need to do so.
+
+Windows (cmd):
+```cmd
+set PATH=%PATH%;C:\your\GOPATH\bin
+```
+
+Bash (you can verify a path has been set):
+```Bash
+where nakedret
```
Comment (alexkohler) [inline README.md:21]: did you mean to do `which nakedtet` here?
```suggestion
# Check if nakedret is on PATH
which nakedret
```

**Review by alexkohler — COMMENTED** [review]
Looks good! Feel free to add an installation gif if you'd like :+1:

**Comment by alexkohler** [comment]
Thanks again!

**Pull request merged**